qds_sdk/commands.py

```diff
@@ -136,6 +150,19 @@ def get_log(self):
         r = conn.get_raw(log_path)
         return r.text
 
+    def get_log_partial(self, err_pointer=0, tmp_pointer=0):
+        """
```

Comment:
I think instead of defining another fn i.e `get_log_partial `, can we use fn `get_log` itself ? 
Maybe modify it to something like:
 `def get_log(self, err_pointer=0, tmp_pointer=0)`
Instead of returning r.text, we now would have to return a list [] and handle it in all places where we are reading the return value from this fn.